evalcache.plugin.zsh

```diff
@@ -7,7 +7,8 @@
 export ZSH_EVALCACHE_DIR=${ZSH_EVALCACHE_DIR:-"$HOME/.zsh-evalcache"}
 
 function _evalcache () {
-  local cacheFile="$ZSH_EVALCACHE_DIR/init-$1.sh"
+  BINARY=$1
```

Comment:
Would it be possible to roll this variable assignment into the local scoped assignment below, e.g. `local cacheFile="$ZSH_EVALCACHE_DIR/init-${1##*/}.sh"`? Or would that break something I'm not anticipating here.

Reply:
Yes, that would work as far as I'm aware.

Reply:
I will update the PR

Reply:
Great, would you mind please updating with that change whenever convenient and verifying it still works for you? (I'd like to keep all variable assignment function-scoped local to avoid any possibility of collisions with the user's shell.)

Reply:
Still works as before